test-case/manual/Audio_privacy.md

```diff
@@ -0,0 +1,48 @@
+# Verify Audio Privacy (Hard Mute) Functionality
+Verify Audio Privacy (Hard Mute) Functionality in Linux
+
+## Preconditions
+1. The RVP should have active audio endpoints.
+2. The RVP should have a functional hard mute switch or toggle.
+3. Enable BIOS settings:
+    3a. Microphone Privacy Mode = "HW managed Microphone Privacy"
+    3b. Enable corresponding codec options, e.g., SNDW#3 [x] for SoundWire.
```

Comment:
4. Make sure the UCM is up to date.

The Audio Privacy needs support by UCM. So, we need to make sure the UCM is up to date before testing.
The UCM repo is https://github.com/alsa-project/alsa-ucm-conf/